webapp/stores/user_store.jsx

```diff
@@ -105,6 +105,10 @@ class UserStoreClass extends EventEmitter {
         this.currentUserId = user.id;
         global.window.mm_current_user_id = this.currentUserId;
         if (LocalizationStore.getLocale() !== user.locale) {
+            if (!GlobalActions) {
```

Comment:
I had to break that dependency when adding the unit tests because we had a dependency chain of TextFormatting > UserStore > GlobalActions > react-router and even importing react-router causes mocha-webpack to freak out. I was going to move the call to GlobalActions to somewhere else, but I couldn't figure out where that should be